apps/builder/app/builder/features/style-panel/style-source/style-source.tsx

```diff
@@ -272,7 +272,9 @@ export const StyleSource = ({
         </StyleSourceButton>
       </Flex>
       {stateLabel !== undefined && (
-        <StyleSourceState>{stateLabel}</StyleSourceState>
+        <StyleSourceState css={{ lineHeight: 1 }}>
```

Comment:
Put it into own style source state styles?

Reply:
can't, because radix resolves variant styles as higher specificity than those, only this way you get to override the variant